packages/babel-plugin-transform-object-rest-spread/src/index.js

```diff
@@ -112,9 +124,15 @@ export default function ({ types: t }) {
           originalPath: path
         });
 
-        if (nodes.length > 0) {
-          path.parentPath.getSibling(path.parentPath.key + 1)
```

Comment:
Not really sure what the point of this line was. Why was it important to `insertBefore` before the next sibling, instead of just `insertAfter` here?